tests/codeinclude/test_languages.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from codeinclude.languages import get_lang_class
+
+
+class MyTestCase(unittest.TestCase):
+    def test_get_lang_class(self):
+        self.assertEquals('java', get_lang_class('HelloWorld.java'))
+        self.assertEquals('python', get_lang_class('HelloWorld.py'))
+        self.assertEquals('csharp', get_lang_class('HelloWorld.cs'))
+        self.assertEquals('rust', get_lang_class('HelloWorld.rs'))
+        self.assertEquals('docker', get_lang_class('Dockerfile'))
+        self.assertEquals('xml', get_lang_class('HelloWorld.xml'))
+        self.assertEquals('toml', get_lang_class('HelloWorld.toml'))
```

Comment:
Hmm, the test seems to fail on this line - I wonder if there's varying support for the `toml` extension in different versions of pygments.

```
>       self.assertEquals('toml', get_lang_class('HelloWorld.toml'))
E       AssertionError: 'toml' != 'none'
E       - toml
E       + none
```

Reply:
It seems I used `Pygments~=2.5` or newer, which one fails? Would it make sense to require a certain version of it, or just remove this test?